setup.cfg

```diff
@@ -47,27 +47,27 @@ database =
   sqlalchemy
 instr =
   matplotlib>=3.1.1
-  pandas>=0.24.0
```

Comment:
according to NEP29 we should be using the version of our deps 24 months before scheduled release which is May 2019 and puts pandas at 0.24. Pandas 1.0 was Jan 2020, so according to our release calendar we wont be there till 4.0.

Reply:
I mean that's right, but pandas went to 1.0.0 and has now a stable API. I feel like bumping to that is a good idea instead of a <1 version for our lts.